package-lock.json

```diff
@@ -4342,13 +4342,6 @@
         "minimalistic-assert": "1.0.0"
       }
     },
-    "design-system": {
```

Comment:
What happened with DS?

Reply:
it's installed via github not registry,
probably this
I'll downgrade npm on my laptop and see what happens